routes/api/comments/index.js

```diff
@@ -148,13 +148,8 @@ router.put('/:comment_id/status', authorization.needed('admin'), (req, res, next
 });
 
 router.post('/:comment_id/actions', (req, res, next) => {
-
-  const {
-    action_type
-  } = req.body;
-
   Comment
-    .addAction(req.params.comment_id, req.user.id, action_type)
+    .addAction(req.params.comment_id, req.user.id, req.body)
```

Comment:
We should extract only the parameters we need from the body, not the whole one, this would result in anyone being able to stuff whatever they want in our db.